test-scripts/tomcat-deployment/jmeter/user.properties

```diff
@@ -0,0 +1 @@
+tomcat_host=http://54.224.223.61:8080
```

Comment:
Who modify this file? Shall we add DO_NOT_MODIFY; AUTO_GENERATED comment at the top? Is this really auto-generated?